### PR TITLE
[ES] Update `Hit_object`

### PR DESCRIPTION
**Pull request by Ekkusu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Hit_object/es.md [hunk 1]
```diff
@@ -1,35 +1,26 @@
...
+tags:
+  - hit objects
+  - objetos de golpeo
 ---
 
-# Hit Objects
+<!-- TODO: should also have articles for each hit object -->
 
-Un *hit object* (objetos de golpeo) es un elemento principal en osu!. Hay tres tipos de *hit objects*:
+<!-- TODO: other games modes? =( -->
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/es.md:10]:
```suggestion
```

File: wiki/Gameplay/Hit_object/es.md [hunk 2]
```diff
@@ -1,35 +1,26 @@
 ---
-outdated_translation: true
+tags:
+  - hit objects
+  - objetos de golpeo
 ---
 
-# Hit Objects
+<!-- TODO: should also have articles for each hit object -->
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/es.md:8]:
```suggestion
```

File: wiki/Gameplay/Hit_object/es.md [hunk 3]
```diff
@@ -1,35 +1,26 @@
...
-
-**Nota:** La actual skin por defecto \[osu! by peppy\] no tiene Approach Circles ni Spinner Metre, la skin por defecto reemplazada \[osu!default by peppy\] los tiene.
+<!-- TODO: Add links to other articles when they're available -->
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/es.md:26]:
```suggestion
```

File: wiki/Gameplay/Hit_object/es.md [hunk 4]
```diff
@@ -1,35 +1,26 @@
...
+Desde el punto de vista de la programación, las notas sostenidas en osu!mania no son el equivalente a los deslizadores de osu!, aunque han sido agrupados juntos para la simplificación de la tabla. Otros elementos con los que no se interactúan durante el juego, como la barra de salud o las estrellas del kiai, se consideran mejoras del juego o parte de la interfaz del usuario.
 
-## Slider
+En osu!, el siguiente circulo de golpeo o deslizador que se toca, se resalta con un brillo ligeramente mayor en comparación de los otros objetos de golpeo en el campo de juego. Sin embargo, el resaltado ya no será mas visible cuando la  [velocidad de aproximación](/wiki/Beatmapping/Approach_rate) sea demasiado alta.
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/es.md:24]:
```suggestion
En osu!, el siguiente círculo de golpeo o deslizador que se toca, se resalta con un brillo ligeramente mayor en comparación de los otros objetos de golpeo en el campo de juego. Sin embargo, el resaltado ya no será más visible cuando la  [velocidad de aproximación](/wiki/Beatmapping/Approach_rate) sea demasiado alta.
```

File: wiki/Gameplay/Hit_object/es.md [hunk 5]
```diff
@@ -1,35 +1,26 @@
...
+| x | x | [hyperfrutas](/wiki/Hit_object/Hyperfruit) | x |
 
-Un *hit circle* en la skin por defecto. Llamada *Hit Marker* en los juegos DS. Es un círculo coloreado con un número en él (dependiendo de su lugar en un Combo) con el contorno de otro círculo (Approach Circle o Círculo de aproximación) encogiéndose alrededor de él. Una vez que el contorno del Aproach Circle se superpone al borde del Hit Circle, el jugador debe pulsar el Hit Circle, consiguiendo un número de puntos (50, 100 o 300) dependiendo de la precisión en la que fue pulsado, y posiblemente anotando un *Beat!* o un *Elite Beat!* si el círculo es el final de un combo. Pulsando *Hit Circles* normales te da un pequeño aumento en la Barra de vida, y un buen aumento si es un círculo de final de combo.
+Desde el punto de vista de la programación, las notas sostenidas en osu!mania no son el equivalente a los deslizadores de osu!, aunque han sido agrupados juntos para la simplificación de la tabla. Otros elementos con los que no se interactúan durante el juego, como la barra de salud o las estrellas del kiai, se consideran mejoras del juego o parte de la interfaz del usuario.
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/es.md:22]:
```suggestion
Desde el punto de vista de la programación, las notas largas en osu!mania no son el equivalente a los deslizadores de osu!, aunque han sido agrupados juntos para la simplificación de la tabla. Otros elementos con los que no se interactúan durante el juego, como la barra de salud o las estrellas del kiai, se consideran mejoras del juego o parte de la interfaz del usuario.
```

File: wiki/Gameplay/Hit_object/es.md [hunk 6]
```diff
@@ -1,35 +1,26 @@
...
+# Objetos de golpeo
 
-Los *Hit circles* y *sliders* son encontrado frecuentemente, mientras que los *spinners* solo aparecen ocasionalmente. Todas las cosas con las que interactuas durante el curso de un beatmap es un *hit object*.
+Los *Objetos de Golpeo* son elementos donde los jugadores pueden interactuar durante el juego de un [mapa](/wiki/Beatmap). Cada [modo de juego](/wiki/Game_mode) tiene su propio conjunto único de objetos de golpeo que cambiaran la experiencia del juego. Para obtener una descripción general de cada modo de juego y sus objetos de golpeo, consulta la siguiente tabla:
 
-## Hit Circle
+| [osu!](/wiki/Game_mode/osu!) | [osu!taiko](/wiki/Game_mode/osu!taiko) | [osu!catch](/wiki/Game_mode/osu!catch) | [osu!mania](/wiki/Game_mode/osu!mania) |
+| :-: | :-: | :-: | :-: |
+| [circulos de golpeo](/wiki/Hit_object/Hit_circle) | circulos de golpeo | [frutas](/wiki/Hit_object/Fruit) | notas |
+| [deslizadores](/wiki/Hit_object/Slider) | redobles de tambor | [gotas de jugo](/wiki/Hit_object/Juice_stream) | notas sostenidas |
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/es.md:18]:
```suggestion
| [deslizadores](/wiki/Hit_object/Slider) | redobles de tambor | [gotas de jugo](/wiki/Hit_object/Juice_stream) | notas largas |
```

**Review by agatemosu — COMMENTED** [review]
I think that's more understandable

File: wiki/Gameplay/Hit_object/es.md [hunk 7]
```diff
@@ -1,35 +1,26 @@
...
-- Spinner
+# Objetos de golpeo
 
-Los *Hit circles* y *sliders* son encontrado frecuentemente, mientras que los *spinners* solo aparecen ocasionalmente. Todas las cosas con las que interactuas durante el curso de un beatmap es un *hit object*.
+Los *Objetos de Golpeo* son elementos donde los jugadores pueden interactuar durante el juego de un [mapa](/wiki/Beatmap). Cada [modo de juego](/wiki/Game_mode) tiene su propio conjunto único de objetos de golpeo que cambiaran la experiencia del juego. Para obtener una descripción general de cada modo de juego y sus objetos de golpeo, consulta la siguiente tabla:
 
-## Hit Circle
+| [osu!](/wiki/Game_mode/osu!) | [osu!taiko](/wiki/Game_mode/osu!taiko) | [osu!catch](/wiki/Game_mode/osu!catch) | [osu!mania](/wiki/Game_mode/osu!mania) |
+| :-: | :-: | :-: | :-: |
+| [circulos de golpeo](/wiki/Hit_object/Hit_circle) | circulos de golpeo | [frutas](/wiki/Hit_object/Fruit) | notas |
```
Comment (agatemosu) [inline wiki/Gameplay/Hit_object/es.md:17]:
```suggestion
| [círculos de golpeo](/wiki/Hit_object/Hit_circle) | círculos de golpeo | [frutas](/wiki/Hit_object/Fruit) | notas |
```

**Comment by Ekkusu** [comment]
> I think that's more understandable

yes, i was thinking a bit, and i think you're right

**Review by agatemosu — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**